src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -414,6 +417,28 @@ public function locale($locale = null)
     }
 
     /**
+     * Returns a fully aliased field name for translated fields.
+     *
+     * If the requested field is configured as a translation field, the `content`
+     * field with an alias of a corresponding association is returned. Table-aliased
+     * field name is returned for all other fields.
+     *
+     * @param string $field Field name to be aliased.
+     * @return string
+     */
+    public function translationField($field)
+    {
+        $table = $this->_table;
+        $associationName = $table->alias() . '_' . $field . '_translation';
+
+        if ($table->associations()->has($associationName)) {
+            return $associationName . '.content';
```

Comment:
Yes, but in that case entire `TranslateBehavior` will encounter issues as this association name is generated the same way multiple times in the code. We could of course add a helper method for generating consistent association names.